sources/engine/Stride.BepuPhysics/Stride.BepuPhysics/Constraints/DistanceServoConstraintComponent.cs

```diff
@@ -11,6 +10,9 @@
 
 namespace Stride.BepuPhysics.Constraints;
 
+/// <summary>
+/// Constrains points on two bodies to be separated by a goal distance.
```

Comment:
Wondering if we should keep the term "target" here, to be consistent with the name of the property.

Reply:
You are right. Thanks for the feedback. I check Bepu's XML comments so I have to make sure that they are reworded according our properties or improved where needed.

Reply:
This is now resolved 🙂